src/SLCore.UnitTests/Listener/Analysis/RaiseFindingToAnalysisIssueConverterTests.cs

```diff
@@ -268,7 +236,7 @@ public void GetAnalysisIssues_HotspotHasNoVulnerabilityProbability_AnalysisHotsp
                 default,
                 null,
                 HotspotStatus.SAFE,
-                default)
+                Either<StandardModeDetails, MQRModeDetails>.CreateLeft(new(default, default)))
```

Comment:
btw, https://github.com/SonarSource/sonarlint-visualstudio/commit/eb24cf22eafb9a400da6b361a8edba8d21eabf88 has added implicit casts from Left/Right to Either<Left, Right>